bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/i18n/netatmo.properties

```diff
@@ -0,0 +1,71 @@
+
```

Comment:
This file needs to be regenerated, seems to be missing all strings from XML definitions.

Reply:
Done